internal/provider/resources/resource_team_test.go

```diff
@@ -74,7 +74,7 @@ func TestAcc_ResourceTeam(t *testing.T) {
 						},
 					},
 				}),
-				ExpectError: regexp.MustCompile(fmt.Sprintf("Role '%s' is not valid for role type '%s'", string(iam.ORGANIZATIONOWNER), string(iam.WORKSPACE))),
+				ExpectError: regexp.MustCompile(".*Invalid Attribute Value Match.*"),
```

Comment:
can you explain the new error msg change? i know that we won't get that original error since the validate config was removed but who sends this/can we modify it?